### ISSUE TITLE
objkt unaccessible for transactions - strange collectors informations shown.

### ISSUE THREAD

**Issue by luluxxxxx** [hicetnunc2000/hicetnunc#926]
it concerns this objkt :  https://www.hicetnunc.xyz/objkt/101177

available number of pieces is -4 x and any interaction leads to an error.

"Parameters Invalid . Some of the parameters you provided are invalid and the request could not be completed. Please check your inputs and try again."
kukai returns this error : "Simulation error: A FAILWITH instruction was reached {"int":"148"}"

people reporting they can't buy.

thanks in advance for your help.
luluxxx

**Comment by davidvnun** [comment]
I got the same issue recently also on this objkt: https://www.hicetnunc.xyz/objkt/124081

I think it has something to do with configuring the name of the wallet on https://www.hicetnunc.xyz/config
This issue started after I did the name configuration. The objkts on my swap were negative and I couldn't interact with them anymore. I had more unswapped objkts and after I listed them that worked but it only listed those, all other swaps remain broken and inaccessible.

I use temple and it simply tells me "The transaction is invalid and the node did not accept it." Whenever I try to swap.
Hope this info is useful in solving this bug.

Thank you!
David
